test/dogma/config_test.exs

```diff
@@ -67,7 +68,7 @@ defmodule Dogma.ConfigTest do
         override: %{ ModuleDoc => [enabled: false] }
       ) do
 
-        stdout = ExUnit.CaptureIO.capture_io(fn->
+        stdout = ExUnit.CaptureIO.capture_io(fn ->
```

Comment:
👍  I can add an option in to allow this form, but I'm thinking enforcing it would be the responsibility of another rule, similar to the range operator `1..2`. I allow no spaces around this operator right now, but I don't enforce it. Do you agree?